app/views/projects/show.html.erb

```diff
@@ -34,7 +34,7 @@
       <p class="fst-italic">There hasn't been any recent workspace activity in this project.</p>
     <% end %>
 
-    <% favorites = @project.workspaces.favorites.accessible_by(current_ability) %>
+    <% favorites = current_user.favorite_workspaces %>
```

Comment:
Check accessibility again or nah?

Reply:
Not sure if this is relevant or not, but if a user favorited a workspace that they then lose access to, I think it would be fine to still keep track of the user's favorite status of that workspace. The user shouldn't see a favorited workspace if they don't have access to it, of course, but if they are later re-granted access to it, it does seem reasonable that upon re-access it is shown to that user as one of their favorite workspaces (since it was before and they never said it wasn't).

This might already be how things work, or might not be possible with the way things are stored in the database, but I just wanted to offer that opinion on the general workings of things.

Reply:
@ggeisler ok, thanks!